pkg/apis/wmc/v1alpha1/windowsmachineconfig_types.go

```diff
@@ -0,0 +1,61 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
+// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+
+// WindowsMachineConfigSpec defines the desired state of WindowsMachineConfig
+type WindowsMachineConfigSpec struct {
+	 // Replicas represent how many Windows nodes to be added to the
+        // OpenShift cluster
+        Replicas int `json:"replicas,omitempty"`
+        // InstanceType represents the flavor of instance to be used while
+        // creating the virtual machines. Please note that this is common
+        // across all the Windows nodes in the cluster
+        InstanceType string `json:"instancetype,omitempty"`
+        // sshkeyPair is the sshKeyPair associated with cloudprovider. AWS
+        // asks a keypair to be present for encrypting the Windows VM password
+        sshkeyPair string `json:"sshkeypair"`
+        // sshPrivateKey is the name of the secret which contains ssh key
+        // to decrypt the password
+        sshPrivateKey string `json:"sshprivatekey"`
+        // cloudProviderCreds is the name of the secret which contains
+        // credentials of the cloud provider.
+        cloudProviderCredentials string `json:"cloudprovidercredentials,omitempty"`
```

Comment:
Please indicate in the variable name that we are using a secret.

Reply:
It would be better to create a struct for AWS as this is just going to grow. We can add a struct for Azure when we start working on adding Azure support.